advice_animal/cli.py

```diff
@@ -188,6 +188,20 @@ def check(ctx: click.Context, target: str) -> None:
             )
 
 
+@main.command()
+@click.pass_context
+def list(ctx: click.Context) -> None:
+    runner = Runner(Path(ctx.obj.advice_path), inplace=False, mode=Mode.check)
+    for advice_name, check_cls in runner.iter_check_classes(
+        FixConfidence.UNSET, True, re.compile(".*")
+    ):
+        if check_cls.confidence.name != "UNSET":
+            name = click.style(advice_name, fg=check_cls.confidence.name.lower())
```

Comment:
nit: we probably ought to have better color handling now that this conveys an entire piece of information.  Here's how hdeps does it: https://github.com/hdeps/hdeps/blob/087fa55a8b772179b11a21aaa8462c8938a2b6c3/hdeps/cli.py#L152-L155